src/_pytest/nodes.py

```diff
@@ -172,6 +172,10 @@ def __init__(
         # own use. Currently only intended for internal plugins.
         self._store = Store()
 
+    @property
+    def qual_name(self) -> str:
+        return "::".join(node.name for node in self.listchain())
```

Comment:
`listnames` exists :) to get a list of names

Reply:
A good we can use that then. 👍

Reply:
Yep, ok to just drop `()` ? maybe we can inspect the parent type(s) to derive if it should be a `/` vs `::`

Reply:
At first glance this seems really strange

That kind of qualname is basically a potentially cut broken nodeid

Reply:
@RonnyPfannschmidt please see #7365, would appreciate your advice there in-line of recent comments

Reply:
I was under the impression that the qualname was supposed to be a python thing, if its going to be a cut up nodeid, it's going to be a problem